ffv1.md

```diff
@@ -803,10 +803,6 @@ SliceHeader( ) {                                              |
     picture_structure                                         | ur
     sar_num                                                   | ur
     sar_den                                                   | ur
-    if (version >= 4) {                                       |
-        reset_contexts                                        | br
```

Comment:
If `reset_contexts` is removed here, there remain a few stranded references to it.

Reply:
Fixed